### ISSUE TITLE
Red bar at the bottom of every page after installing

### ISSUE THREAD

**Issue by shaqaruden** [kenepa/resource-lock#10]
After installing Resource Lock there is a red bar at the bottom of every page.
![image](https://user-images.githubusercontent.com/5414897/234856843-99445d90-3340-4405-9bfc-f4adc24af80c.png)

When inspecting the HTML I know this is from this package

```html
<div wire:id="DVMC6eW3Ywy82NGFmshx" x-init="resourceLockObserverInit" class="bg-red-500">

    <script>
        function resourceLockObserverInit() {
            Livewire.emit('resourceLockObserver::init')
        }

        window.onbeforeunload = function () {
            Livewire.emit('resourceLockObserver::unload')
        };

        window.addEventListener('close-modal', event => {
            if (event.detail.id.endsWith('-table-action')) {
                Livewire.emit('resourceLockObserver::unload')
            }
        })
    </script>

    <div x-data="{

        isOpen: false,

        livewire: null,

        close: function () {
            this.isOpen = false

            this.$refs.modalContainer.dispatchEvent(new CustomEvent('modal-closed', { id: 'resourceIsLockedNotice' }))
        },

        open: function () {
            this.isOpen = true

            this.$refs.modalContainer.dispatchEvent(new CustomEvent('modal-opened', { id: 'resourceIsLockedNotice' }))
        },

    }" x-trap.noscroll="isOpen" x-on:close-modal.window="if ($event.detail.id === 'resourceIsLockedNotice') close()" x-on:open-modal.window="if ($event.detail.id === 'resourceIsLockedNotice') open()" role="dialog" aria-modal="true" class="filament-modal inline-block" wire:ignore.self="">
    

    <div x-show="isOpen" x-transition.duration.300ms.opacity="" class="fixed inset-0 z-40 min-h-full overflow-y-auto overflow-x-hidden transition flex items-center" style="display: none;">
        <div aria-hidden="true" class="filament-modal-close-overlay fixed inset-0 w-full h-full bg-black/50"></div>

        <div x-ref="modalContainer" class="relative w-full cursor-pointer pointer-events-none transition my-auto p-4" disabled="disabled">
            <div x-data="{ isShown: false }" x-init="$nextTick(()=> {
                    isShown = isOpen
                    $watch('isOpen', () => isShown = isOpen)
                })" x-show="isShown" x-on:keydown.window.escape="$dispatch('close-modal', { id: 'resourceIsLockedNotice' })" x-transition:enter="ease duration-300" x-transition:leave="ease duration-300" x-transition:enter-start="translate-y-8" x-transition:enter-end="translate-y-0" x-transition:leave-start="translate-y-0" x-transition:leave-end="translate-y-8" class="filament-modal-window w-full py-2 bg-white cursor-default pointer-events-auto dark:bg-gray-800 relative rounded-xl mx-auto max-w-sm" style="display: none;">
                
                <div class="">
                    <div class="space-y-2">
                        
                                            </div>

                    <div class="filament-modal-content space-y-2 p-2">
                        
                                                    <div class="px-4 py-2 space-y-4">
                                <div x-data="{ resourceLockOwner: null}" @open-modal.window="(event) => { resourceLockOwner = event.detail.resourceLockOwner}">
            <div class="flex justify-center ">
                <button type="button" class="filament-icon-button flex items-center justify-center rounded-full relative outline-none hover:bg-gray-500/5 disabled:opacity-70 disabled:cursor-not-allowed disabled:pointer-events-none text-primary-500 focus:bg-primary-500/10 dark:hover:bg-gray-300/5 w-12 h-12">
        
        <svg wire:loading.remove.delay="" wire:target="" class="filament-icon-button-icon w-6 h-6" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
  <path fill-rule="evenodd" d="M5 9V7a5 5 0 0110 0v2a2 2 0 012 2v5a2 2 0 01-2 2H5a2 2 0 01-2-2v-5a2 2 0 012-2zm8-2v2H7V7a3 3 0 016 0z" clip-rule="evenodd"></path>
</svg>
        
            </button>
            </div>
            <p x-show="resourceLockOwner" class="text-center" style="display: none;">
                <span x-text="resourceLockOwner" class="font-bold"></span> is currently editing this page, so it is currently locked for editing.
            </p>
            <p x-show="resourceLockOwner === null" class="text-center">
                This page has been locked because another user is currently editing it.
            </p>
        </div>

        <div x-data="{url: '/'}" @open-modal.window="(event) => { url = event.detail.returnUrl}" class="flex flex-col justify-center space-y-2">

            
            <a class="block filament-button filament-button-size-md inline-flex items-center justify-center py-1 gap-1 font-medium rounded-lg border transition-colors outline-none focus:ring-offset-2 focus:ring-2 focus:ring-inset min-h-[2.25rem] px-4 text-sm text-white shadow focus:ring-white border-transparent bg-primary-600 hover:bg-primary-500 focus:bg-primary-700 focus:ring-offset-primary-700 filament-page-button-action" :href="url" href="/">

            <span>
                Return
            </span>

            </a>
        </div>
                            </div>
                        
                        
                    </div>

                    <div class="space-y-2">
                        
                                            </div>
                </div>
            </div>
        </div>
    </div>
</div>
</div>
```

**Comment by Jehizkia** [comment]
Thank you, will be fixed soon!

**Comment by Jehizkia** [comment]
A new release has been made, it should be fixed in version  [1.1.1](https://github.com/kenepa/resource-lock/releases/tag/1.1.1)

**Comment by shaqaruden** [comment]
Wow that was fast!
